docs/howto/configure/auth-management.md

```diff
@@ -185,7 +185,8 @@ There are also some steps that a Community Representative will need to take to s
 Once the OAuth callbacks have been set up following the steps above, somebody with administrator permissions for the GitHub organization needs to grant access to the OAuth app that we have created.
 
 The first time that somebody tries to log on to the hub with this authentication set up, they should be presented with a page that asks them to grant access to various GitHub organizations.
-**They must grant access to the organization that is being used to authenticate**.
+For each user, GitHub will list _all organizations to which they can control authorization_.
+**They must grant access to the organization that is added to `allowed_organizations` above**, but do not need to grant access to any other organizations.
```

Comment:
Let's note that 'granting access' just lets us know they belong to the org? We don't get much other access